app/apollo/test/data/local/fga.spec.user02.json

```diff
@@ -5,30 +5,30 @@
   "orgName": "org_01",
   "authorization": {
     "cluster": {
-      "read": "test-cluster2-uuid",
-      "attach": "test-cluster2-uuid",
-      "detach": "test-cluster2-uuid",
-      "register": "test-cluster2-uuid",
-      "update": "test-cluster2-uuid"
+      "read": ["test-cluster2-uuid"],
+      "attach": ["test-cluster2-uuid"],
+      "detach": ["test-cluster2-uuid"],
+      "register": ["test-cluster2-uuid"],
+      "update": ["test-cluster2-uuid"]
     },
     "group": {
-      "manage": "test-group2-uuid",
-      "read": "test-group2-uuid",
-      "setversion": "test-group2-uuid"
+      "manage": ["test-group2-uuid", "test-add-group-uuid"],
+      "read": ["test-group2-uuid", "test-add-group-uuid"],
+      "setversion": ["test-group2-uuid", "test-add-group-uuid"]
```

Comment:
**_real_** uuids of created resources can't be known ahead of time.  The new group added by tests should be referred to by _name_, e.g.
```
      "manage": ["test-group2-uuid", "test-add-group-name"],
      "read": ["test-group2-uuid", "test-add-group-name"],
      "setversion": ["test-group2-uuid", "test-add-group-name"]
```
This helps demonstrate and test that auth by uuid _**or**_ name is allowed (though that's up to the auth model internals)

Reply:
makes sense! fixed

Reply:
resolved